SynCrtSock.pas

```diff
@@ -1843,6 +1845,11 @@   THttpServer = class(THttpServerGeneric)
     // a THttpServerResp thread is created for handling this THttpServerSocket
     property Sock: TCrtSocket read fSock;
   published
+    /// flag to disable HTTP/1.1 keep alive features and fall back to
+    // HTTP/1.0 compatibility
+    // Only threads from thread pool will be used to process requests
+    // when this flag set to true
+    property DisableKeepAliveSupport: boolean read fDisableKeepAliveSupport;

```

Comment:
add "write" attribute too?

Reply:
Let's don't forget that this is a thread so execution starts immediately. It could be too late to set this flag after execution started and first requests came
On the other side, IMHO it is non-practical to switch such behavior on the fly.